Signal/src/util/OWSBackupImportJob.m

```diff
@@ -329,29 +365,28 @@ - (void)restoreDatabase:(OWSBackupJobBoolCompletion)completion
         }
         return databaseKeySpec;
     };
-    OWSBackupStorage *_Nullable backupStorage =
```

Comment:
`backup storage` reference should hang on the job, so that we can dealloc it synchronously.